sqlite/src/func.c

```diff
@@ -2700,6 +2700,7 @@ void sqlite3RegisterBuiltinFunctions(void){
     VFUNCTION(guid_str,          0, 0, 0, guidStrFunc      ),
     FUNCTION(guid,               1, 0, 0, guidFromStrFunc  ),
     FUNCTION(guid_str,           1, 0, 0, guidFromByteFunc ),
+    VFUNCTION(libuuid_call,      0, 0, 0, guidFunc         ),
```

Comment:
I don't get the point behind adding this one. We already have 'guid' vfunc registered above.